python/src/com/jetbrains/python/psi/types/PyModuleType.java

```diff
@@ -368,21 +368,35 @@ private static CompletionVariantsProcessor createCompletionVariantsProcessor(Psi
                                                                               @NotNull PsiElement location,
                                                                               @Nullable final Set<String> existingNames) {
 
-    final List<PsiElement> elements = collectImportedSubmodules(pyPackage, location);
-    return elements != null ? ContainerUtil.mapNotNull(elements,
-                                                       (Function<PsiElement, LookupElement>)element -> {
-                                                         if (element instanceof PsiFileSystemItem) {
-                                                           return buildFileLookupElement((PsiFileSystemItem)element, existingNames);
-                                                         }
-                                                         else if (element instanceof PsiNamedElement) {
-                                                           return LookupElementBuilder.createWithIcon((PsiNamedElement)element);
-                                                         }
-                                                         return null;
-                                                       }) : Collections.emptyList();
+    List<LookupElement> lookupElements = new SmartList<>();
+    processImportedSubmodules(pyPackage, location, (importElement, importedSubmodule) -> {
+      if (importedSubmodule instanceof PsiFileSystemItem) {
+        lookupElements.add(buildFileLookupElement((PsiFileSystemItem)importedSubmodule, existingNames));
+      }
+      else if (importedSubmodule instanceof PsiNamedElement) {
+        lookupElements.add(LookupElementBuilder.createWithIcon((PsiNamedElement)importedSubmodule));
+      }
+      return true;
+    });
+    return lookupElements;
   }
 
-  @Nullable
+  @Deprecated
```

Comment:
See how we deprecate methods in PyCallExpression, there are notes about version to remove and alternative way to achieve the result.

Reply:
Which version should I put down?

Reply:
2018.3